src/Our.Umbraco.Gandalf/Core/StartUp/IpLockerComposer.cs

```diff
@@ -2,18 +2,39 @@
 using Umbraco.Core.Composing;
 using Our.Umbraco.Gandalf.Core.Repositories;
 using Our.Umbraco.Gandalf.Core.Services;
+using System.Configuration;
+using System.ComponentModel;
+using DangEasy.Interfaces.Caching;
+using DangEasy.Caching.MemoryCache;
+using Umbraco.Core;
+using Our.Umbraco.Gandalf.Core.Services.CachedProxies;
 
 namespace Our.Umbraco.Gandalf.Core.StartUp
 {
 	public class GandalfComposer : IUserComposer
 	{
+
 		public void Compose(Composition composition)
 		{
-			composition.Register(typeof(IAllowedIpService), typeof(AllowedIpService));
+			composition.Register(typeof(ICache), typeof(Cache));
+
```

Comment:
This package should always use caching IMO. We don't really want people to turn this off. However we need to turn it off for our own testing.

Perhaps if the appsetting does NOT exist, then assume cache is turned on.

If you WANT to turn it off, then add the app setting = false